source/educators/olx/example-course/index.rst

```diff
@@ -22,3 +24,9 @@ courseware is defined.
 
    insider-structure
    insider-course-xml
+
```

Comment:
do we need a "**Maintenance chart**" line here?

Reply:
I added one because I wanted to call out that the edx-insider course is very old and not in the Open edX org

Reply:
Sorry, I wasn't clear. I definitely agree with adding the chart, but I think it's missing the header: 

```
**Maintenance chart**
```

Reply:
Ha! Yes, I'll add that in.

```suggestion

**Maintenance Chart**


```